node-repository/src/main/java/com/yahoo/vespa/hosted/provision/restapi/ApplicationSerializer.java

```diff
@@ -71,12 +71,12 @@ private static void toSlime(Application application,
         if (cluster.shouldSuggestResources(currentResources))
             cluster.suggestedResources().ifPresent(suggested -> toSlime(suggested.resources(), clusterObject.setObject("suggested")));
         cluster.targetResources().ifPresent(target -> toSlime(target, clusterObject.setObject("target")));
-        clusterUtilizationToSlime(clusterModel, clusterObject.setObject("utilization"));
+        clusterModel.ifPresent(model -> clusterUtilizationToSlime(model, clusterObject.setObject("utilization")));
         scalingEventsToSlime(cluster.scalingEvents(), clusterObject.setArray("scalingEvents"));
         clusterObject.setString("autoscalingStatus", cluster.autoscalingStatus());
-        clusterObject.setLong("scalingDuration", clusterModel.scalingDuration().toMillis());
-        clusterObject.setDouble("maxQueryGrowthRate", clusterModel.maxQueryGrowthRate());
-        clusterObject.setDouble("currentQueryFractionOfMax", clusterModel.queryFractionOfMax());
+        clusterModel.ifPresent(model -> clusterObject.setLong("scalingDuration", model.scalingDuration().toMillis()));
```

Comment:
This + the 2 field below will cause NPE in controller-api/ClusterData if not set

Reply:
thanks!